skiko/src/nativeJsMain/cpp/ImageFilter.cc

```diff
@@ -6,6 +6,8 @@
 #include "SkRect.h"
 #include "common.h"
 
+using SkRuntimeShaderBuilder = SkRuntimeEffectBuilder;
```

Comment:
In Jvm bindings it was replaced directly to SkRuntimeEffectBuilder. 
Is it possible to replace here too, for consistency?

Reply:
@eymar sure, it just that introducing alias was minimal intrusion but you are right - it's better to be homomorphic in that aspect, so let me do it